src/components/accession2/Accession2Create.tsx

```diff
@@ -0,0 +1,176 @@
+  const { organization } = props;
+  const { isMobile } = useDeviceInfo();
+  const theme = useTheme();
+  const history = useHistory();
+  const [record, setRecord, onChange] = useForm<AccessionPostRequestBody>(defaultAccession());
+  const [dates, setDates] = useState<Dates>({
+    collectedDate: record.collectedDate,
+  });
+  const [dateErrors, setDateErrors] = useState<DateErrors>({});
+  const accessions2Database = {
+    pathname: APP_PATHS.ACCESSIONS2,
+  };
+
+  const marginTop = {
+    marginTop: theme.spacing(2),
+  };
+
+  const datePickerStyle = {
+    '.MuiFormControl-root': {
+      width: '100%',
+    },
+    ...marginTop,
+  };
+
+  const changeDate = (id: string, value?: any) => {
+    setDates((curr) => ({ ...curr, collectedDate: value }));
+    const date = new Date(value).getTime();
+    const now = Date.now();
+    if (isNaN(date) || date > now) {
+      setDateErrors((curr) => ({ ...curr, [id]: true }));
+    } else {
+      setDateErrors((curr) => ({ ...curr, [id]: false }));
+      onChange(id, value);
+    }
+  };
+
+  const goToAccessions2 = () => {
+    history.push(accessions2Database);
+  };
+
+  const saveAccession = async () => {
+    try {
+      const accessionNumber = await postAccession(record);
+      history.replace(accessions2Database);
+      history.push({
+        pathname: APP_PATHS.ACCESSIONS2_ITEM.replace(':accessionId', accessionNumber),
+      });
+    } catch (e) {
+      // show toast error
+    }
+  };
+
+  const gridSize = () => (isMobile ? 12 : 6);
+
+  return (
+    <Box
+      display='flex'
+      flexDirection='column'
+      marginTop={theme.spacing(5)}
+      marginBottom={isMobile ? theme.spacing(15) : theme.spacing(5)}
+    >
+      <Typography variant='h2' sx={{ fontSize: '24px', fontWeight: 'bold', margin: '0 auto' }}>
+        {strings.ADD_AN_ACCESSION}
+      </Typography>
+      <Container
+        maxWidth={false}
+        sx={{
+          margin: '0 auto',
+          width: isMobile ? '100%' : '640px',
+          paddingLeft: theme.spacing(4),
+          paddingRight: theme.spacing(4),
+          paddingTop: theme.spacing(5),
+          paddingBottom: theme.spacing(5),
+        }}
+      >
+        <Grid container>
+          <Grid item xs={12}>
+            <Typography variant='h2' sx={SubTitleStyle}>
+              {strings.SEED_COLLECTION_DETAIL}
+            </Typography>
+          </Grid>
+          <Grid item xs={12} sx={marginTop}>
+            <Species2Dropdown record={record} organization={organization} setRecord={setRecord} />
+          </Grid>
+          <Grid item xs={12} sx={datePickerStyle}>
+            <DatePicker
+              id='collectedDate'
+              label={strings.COLLECTED_DATE_REQUIRED}
+              aria-label={strings.COLLECTED_DATE_REQUIRED}
+              value={dates.collectedDate}
+              onChange={changeDate}
+              error={dateErrors.collectedDate === true}
```

Comment:
What does this error do in the UI? I tested and I don't see anything

Reply:
Yeah.. it doesn't work, that's sad. I hoped it would, same for maxDate property. I will create a Jira for us to clean up Date picker properties (make them work or remove them). I will remove this code here for now. Thanks for pointing that out.